.github/CHANGELOG.md

```diff
@@ -1,6 +1,49 @@
 # Release 0.16.0-dev (development release)
 
 <h3>New features since last release</h3>
+
+* PennyLane now has a ``kernels`` module.
+  It provides basic functionalities for working with quantum kernels as well as 
+  post-processing methods to mitigate sampling errors and device noise:
+
+```python
+import pennylane as qml
+from pennylane import numpy as np
+
+num_wires = 6
+wires = range(num_wires)
+
+dev = qml.device('default.qubit', wires=num_wires)
+
+@qml.qnode(dev)
+def kernel_circuit(x1, x2):
+    qml.templates.AngleEmbedding(x1, wires=wires)
+    qml.adjoint(qml.templates.AngleEmbedding(x2, wires=wires))
```

Comment:
@dwierichs  is this not wrong syntax, thinking of your issue #1336 ? 

+1 reason for making this work :)

Reply:
Haha I thought I caught all of them. Thanks!